test/astar.cc

```diff
@@ -1102,6 +1102,141 @@ void test_time_restricted_road_allowed_on_timedep() {
   }
 }
 
+Api timed_access_restriction(std::string mode, std::string datetime) {
+  // The restriction is <tag k="bicycle:conditional" v="no @ (Su 08:00-18:00)"/>
+  // and <tag k="motor_vehicle:conditional" v="no @ (Su 08:00-18:00)"/>
+  auto conf = get_conf("ny_ar_tiles");
+  route_tester tester(conf);
+  std::string request =
+      R"({
+            "locations":[{"lat":40.71835519823214,"lon":-73.99010449658817},{"lat":40.72136384343179,"lon":-73.98817330609745}],
+            "costing":")" +
+      mode + R"(",
+              "date_time":{
+                "type":1,
+                "value":")" +
+      datetime + R"("
+          }
+        })";
+  return tester.test(request);
+}
+
+// The following requests results in timedep astar during the non-restricted hours
+// and should be allowed
+void test_timed_no_access_restriction_1() {
+  auto response = timed_access_restriction("bicycle", "2018-05-13T19:14");
+  const auto& legs = response.trip().routes(0).legs();
+  const auto& directions = response.directions().routes(0).legs(0);
+  const auto& maneuvers_size = directions.maneuver_size();
+  if (maneuvers_size != 3) {
+    throw std::logic_error("This route should remain on Orchard St.");
```

Comment:
This error message seems different from the assertion that there should be 3 maneuvers?